examples/MvcWeb/TheraLang/Controllers/PaginationController.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using MvcWeb.Db;
+using MvcWeb.TheraLang.Entities;
+using MvcWeb.TheraLang.Pagination;
+namespace MvcWeb.TheraLang.Controllers
+{
+    [Route("api/pag/projects")]
+    [ApiController]
+    public class PaginationController : Controller
+    {
+        IttmmDbContext context;
+        public PaginationController(IttmmDbContext Context)
```

Comment:
don't use DbContext in controller, please!

Reply:
ohh sry,  dsdn`t notice

Reply:
> don't use DbContext in controller, please!

ohh sry, dsdn`t notice